rviz_common/src/rviz_common/visualization_frame.hpp

```diff
@@ -526,6 +526,8 @@ protected Q_SLOTS:
 
   /// Indicates if the toolbar should be visible outside of fullscreen mode.
   bool toolbar_visible_;
+
+  std::string node_name_;
```

Comment:
This seems unrelated. Maybe it was supposed to be part of another pr? I'll not let this postpone merging this, but I'll let you guys follow up if necessary.

Reply:
The `node_name_` member is required for handing the node name over to the PanelFactory constructor that is only called in the `initialize` and not in the VisualizationFrame constructor.